gneiss/regression/_ols.py

```diff
@@ -298,3 +318,144 @@ def r2(self):
 
         sst = sse + ssr
         return 1 - sse / sst
+
+    @property
+    def mse(self):
```

Comment:
I'll add it in.  It is a measure of Mean of Sum of Squares Error, which is used for calculating R^2 and the cross validation statistics.